01 - Community/People/javalent.md

```diff
@@ -0,0 +1,66 @@
+---
+aliases:
+- Jeremy Valentine
+tags:
+- 
+publish: true
+---
+
+# Jeremy Valentine
+
+- GitHub: [javalent](https://github.com/javalent/) ^github
```

Comment:
It seems that Jeremy created a "community" page for his plugins, so this is correct.